auction-app-backend/src/main/java/com/example/auctionapp/repository/TestRepository.java

```diff
@@ -0,0 +1,6 @@
+package com.example.auctionapp.repository;
+
+import com.example.auctionapp.entity.TestEntity;
+import org.springframework.data.repository.CrudRepository;
+
+public interface TestRepository extends CrudRepository<TestEntity, Integer> {}
```

Comment:
We should extend `JpaRepository`, becasue it contains everything from `CrudRepository`, but also supports paging and sorting.